raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -3,22 +3,30 @@ pragma solidity ^0.4.0;
 import "./Token.sol";
 
 library NettingChannelLibrary {
-    struct Lock
-    {
-        uint64 expiration;
-        uint amount;
-        bytes32 hashlock;
-    }
-
     struct Participant
     {
         address node_address;
+
+        // Total amount of token transferred to this smart contract through the
+        // `deposit` function, note that direct token transfer cannot be
+        // tracked and will be burned.
         uint256 balance;
-        uint256 netted;
+
+        // The latest known merkle root of the pending hash-time locks, used to
+        // validate the withdrawn proofs.
+        bytes32 locksroot;
+
+        // The latest known transferred_amount from this node to the other
+        // participant, used to compute the netted balance on settlement.
```

Comment:
We should fix that all over the code at some point, but let's at least not try to introduce it more times:
`netted` -> `net`